config/default/kustomization.yaml

```diff
@@ -33,6 +37,28 @@ patchesStrategicMerge:
 - inferenceservice_conversion_webhook.yaml
 - cainjection_conversion_webhook.yaml
 
+patches:
+  # Since OpenShift serving-certificates are being used,
+  # remove CA bundle placeholders
+  - patch: |-
```

Comment:
do we need to delete this? we can add `service.beta.openshift.io/inject-cabundle: "true"` then the ca will be updated automatically. https://docs.openshift.com/container-platform/4.8/security/certificates/service-serving-certificate.html#add-service-certificate-mutating-webhook_service-serving-certificate

Reply:
These are here just because it replicates what we had in the `odh-manifests` PR.

That said, I'm happy to remove these patches if are unneeded.